src/test/safe.dom.settings.owners.test.js

```diff
@@ -142,15 +157,62 @@ describe('DOM > Feature > Settings - Manage owners', () => {
 
     fireEvent.click(SafeDom.getByTestId(ADD_OWNER_THRESHOLD_NEXT_BTN_TESTID))
     await sleep(200)
-    fireEvent.click(SafeDom.getByTestId(ADD_OWNER_SUBMIT_BTN))
+    fireEvent.click(SafeDom.getByTestId(ADD_OWNER_SUBMIT_BTN_TESTID))
     await sleep(1000)
 
     // check if owner was added
-    const newOwnerRow = SafeDom.getAllByTestId(OWNERS_ROW_TESTID)[1]
-
-    expect(newOwnerRow).toHaveTextContent(NEW_OWNER_NAME)
-    expect(newOwnerRow).toHaveTextContent(NEW_OWNER_ADDRESS)
+    ownerRows = SafeDom.getAllByTestId(OWNERS_ROW_TESTID)
+    expect(ownerRows.length).toBe(2)
+    expect(ownerRows[0]).toHaveTextContent('Adol 1 Eth Account0x90F8bf6A479f320ead074411a4B0e7944Ea8c9C1')
+    expect(ownerRows[1]).toHaveTextContent(`${NEW_OWNER_NAME}${NEW_OWNER_ADDRESS}`)
   })
 
-  it('Replaces a owner', async () => {})
+  it('Replaces an owner', async () => {
+    const NEW_OWNER_NAME = 'I replaced an old owner'
+    const NEW_OWNER_ADDRESS = '0x1dF62f291b2E969fB0849d99D9Ce41e2F137006e'
+
+    const twoOwnersSafeAddress = await aMinedSafe(store, 2)
+
+    const SafeDom = renderSafeView(store, twoOwnersSafeAddress)
+    await sleep(1300)
+
+    // Travel to settings
+    const settingsBtn = SafeDom.getByTestId(SETTINGS_TAB_BTN_TESTID)
+    fireEvent.click(settingsBtn)
+    await sleep(200)
+
+    // click on owners settings
+    const ownersSettingsBtn = SafeDom.getByTestId(OWNERS_SETTINGS_TAB_TESTID)
+    fireEvent.click(ownersSettingsBtn)
+    await sleep(200)
+
+    // check if there are 2 owners
+    let ownerRows = SafeDom.getAllByTestId(OWNERS_ROW_TESTID)
+    expect(ownerRows.length).toBe(2)
+    expect(ownerRows[0]).toHaveTextContent('Adol 1 Eth Account0x90F8bf6A479f320ead074411a4B0e7944Ea8c9C1')
```

Comment:
it should work just with `.toHaveTextContent('0x90F8bf6A479f320ead074411a4B0e7944Ea8c9C1')`. This way it's too fragile

Reply:
There were issues where the name of some owners became "unknown", so I feel confident testing addresses and names.

Reply:
But if we change the order of columns it will break. I think it would be better to split these in 3 parts (toHaveTextContent(name), toHaveTextContent(address)).